squid/metadata.csv

```diff
@@ -0,0 +1,53 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+squid.client_http.requests,gauge,,requests,second,The number of HTTP requests received from clients,0,squid,client http reqs
```

Comment:
I almost made a comment about these being `rate`s, then I remembered the in-app type is a `gauge`! 🍰